tools/CouchDB_Import_Instruments.php

```diff
@@ -116,7 +116,7 @@ function generateDocumentSQL(string $tablename) : string
 
         // add the SQL table to the query
         $extraSelect = ", i.* ";
-        $extraJoin = "JOIN $tablename i ON (i.CommentID=f.CommentID) ";
+        $extraJoin = "JOIN ".$this->SQLDB->escape($tablename)." i ON (i.CommentID=f.CommentID) ";
```

Comment:
```suggestion
        $extraJoin = "JOIN " . $this->SQLDB->escape($tablename) . " i ON (i.CommentID=f.CommentID) ";
```

Maybe it's just the way GitHub is highlighting it, but this looks weird (it looks like the dots are inside the string surrounding the table name, even though they're just the PHP concatenation operator outside of quotes.)